docs/FAQ.md

```diff
@@ -175,6 +179,8 @@ directory.
 
 We hope to integrate with Gerrit natively in the future.
 
+[operator]: revsets.md#operators
```

Comment:
Nit: the links are sorted alphabetically, so move it below.

Reply:
Ahhhh, I was trying to figure out what that ordering was. 👍🏼